src/test/java/com/nirima/jenkins/plugins/docker/DockerCloudTest.java

```diff
@@ -58,34 +58,14 @@ public void testConstructor_0_10_2() {
 
     private static final String LOG_MESSAGE = "Docker cloud requires a non-blank name after Jenkins 2.402";
 
-    @Issue("JENKINS-70729") // Warn if cloud name is empty
-    @Test
-    public void testConstructorWithEmptyName() {
-        lr.record(DockerCloud.class.getName(), Level.ALL).capture(16);
-        DockerCloud cloud =
-                new DockerCloud("", new DockerAPI(new DockerServerEndpoint("uri", "credentialsId")), List.of());
-        Assert.assertEquals(cloud.getDisplayName(), "");
-        MatcherAssert.assertThat(lr.getMessages(), IsIterableContaining.hasItem(LOG_MESSAGE));
-    }
-
-    @Issue("JENKINS-70729") // Warn if cloud name is null
-    @Test
-    public void testConstructorWithNullName() {
-        lr.record(DockerCloud.class.getName(), Level.ALL).capture(16);
-        DockerCloud cloud =
-                new DockerCloud(null, new DockerAPI(new DockerServerEndpoint("uri", "credentialsId")), List.of());
-        Assert.assertEquals(cloud.getDisplayName(), null);
-        MatcherAssert.assertThat(lr.getMessages(), IsIterableContaining.hasItem(LOG_MESSAGE));
-    }
-
     @Issue("JENKINS-70729") // Handle null or empty cloud name
     @Test
     public void testCopyConstructor() {
         lr.record(DockerCloud.class.getName(), Level.ALL).capture(16);
         DockerCloud cloud =
-                new DockerCloud(null, new DockerAPI(new DockerServerEndpoint("uri", "credentialsId")), List.of());
+                new DockerCloud("tmp", new DockerAPI(new DockerServerEndpoint("uri", "credentialsId")), List.of());
```

Comment:
@MarkEWaite I kept the CopyConstructor test since it covers backwards compatibility, but I had to modify it to allow an "empty" named DockerCloud

Reply:
Thanks @car-roll .  That makes sense to me.  Assigning directly to the `name` attribute is a nice technique.  I would not have thought of that.